third_party/move/tools/move-cli/src/base/test.rs

```diff
@@ -268,6 +269,10 @@ pub fn run_move_unit_tests<W: Write + Send>(
 
     // Compute the coverage map. This will be used by other commands after this.
     if compute_coverage && !no_tests {
+        if *TRACING_ENABLED {
```

Comment:
@brmataptos I add flush before the trace file is used for generating the coverage map.